doc/object_properties.adoc

```diff
@@ -0,0 +1,265 @@
+
+=== `pointable`
+* field type: `bool`
+
+Whether the object can be pointed at.
+
+=== `visual`
+* field type: `string`
+
+Possible values:
+
+* `"cube"`: A cube that has the same size as a normal node
+* `"item"`: similar to `wielditem`, but ignores the `wield_item` property
+* `"mesh"`: Use a mesh model, see the `mesh` property 
+* `"sprite"`: A texture that is flat and always faces the player
+* `"upright_sprite"`: A texture that is flat and faces upright
+* `"wielditem"`: Used for dropped items, see the `wield_item` property
+
+What the object will look like.
+
+TIP: The `"wielditem"` value supports hardware colorization
+
+=== `mesh`
+* field type: `string`
+
+Name of the file of the mesh, only when using "mesh" visual.
+
+=== `visual_size`
+* field type: `vector`
+
+Visual size multipliers. Wanted value is `{x = x, y = y, z = z}`, if `z` is not provided, `x` will be used for the `z` value.
+
+=== `textures`
+* field type: `string list`
+
+Wanted value is `{"texture.png", "texture.png", ...}`. The number of textures depends on `visual`:
+
+* "cube": 6 textures
+* "mesh": 1 texture for each mesh material
+* "sprite": 1 texture
+* "upright_sprite": 2 textures
+
+=== `colors`
+* internal type: `ColorSpec table`
+
+WARNING: This feature is not functional.
+
+=== `spritediv`
+* field type: `2d_vector`
+
+Determines the number of columns, rows in a spritesheet texture, which is used for animations.. Wanted value is `{x = columns, y = rows}`.
+
+=== `initial_sprite_basepos`
+* field type: `2d_vector`
+
+Determines the position of the first frame to use in the spritesheet texture. Wanted value is `{x = column, y = row}`.
+
+=== `is_visible`
+* field type: `bool`
+
+Whether the object is visible and can be pointed at.
+
+=== `makes_footstep_sound`
+* field type: `bool`
+
+Whether the object makes a footstep sound when it steps on nodes that that have footstep sounds.
+
+=== `stepheight`
+* internal type: `f32`
+
+How high in nodes the object can step up.
+
+=== `automatic_rotate`
+* internal type: `f32`
+
+How fast the object automatically rotates along the `y` axis in radians/second.
+
+=== `automatic_face_movement_dir`
+* field type: `int` or `false`
+* unit: `degrees`
+
+TIP: This property can be set to `false` to disable it.
+
+Make the object automatically face the direction it is moving in, the value is the offset in degrees.
+
+=== `backface_culling`
+* field type: `bool`
+
+Whether a model is backface culled (the faces of the model who's backs are towards a player aren't rendered).
+
+=== `glow`
+* field type: `int`
+
+When calculating the texture color, add this much lighting. Maximum value is `minetest.LIGHT_MAX`.
+
+=== `nametag`
+* field type: `string`
+
+Text displayed about an object, empty by default for all objects except players, in which case, it is their name.
+
+=== `nametag_color`
+* internal type: `ColorSpec`
+
+The color that the nametag text will be.
+
+TIP: This property can be set to `false` to disable it.
```

Comment:
The string"false" or the `bool` value `false`? If the latter, it should be "`ColorSpec` or `bool`.